### PR TITLE
added check for download path for new msi versions

### PR DESCRIPTION
**Pull request by drekuru**
Added Path validation for install location
Updated docs semver link as well

### CODE REVIEW

**Comment by drekuru** [comment]
@aaronpowell 
Seems your actions are failing here

```
Run Install-Module -Force -Scope CurrentUser Pester -MaximumVersion '5.99.99'
WARNING: The version '5.4.1' of module 'Pester' is currently in use. Retry the operation after closing the applications.
No usable version of the libssl was found
Error: Process completed with exit code 1.```

File: nvm.psm1 [hunk 1]
```diff
@@ -371,7 +371,18 @@ function Install-NodeVersion {
                 throw $errMsg
             }
 
-            Move-Item (Join-Path (Join-Path $unpackPath 'nodejs') '*') -Destination $versionPath -Force
+            # build expected path to nodejs
+            $expectedPath = Join-Path $unpackPath 'nodejs'
+
+            if (-Not (Test-Path $expectedPath)) {
+                $expectedPath = Join-Path (Join-Path $unpackPath 'PFiles64') 'nodejs'
```
Comment (aaronpowell) [inline nvm.psm1]: We need to check if the folder is `PFiles` or `PFiles64`, depending on the architecture of the install (64bit vs 32bit).

Can you expand this check to factor that in.

**Review by aaronpowell — CHANGES_REQUESTED** [review]

**Comment by aaronpowell** [comment]
> @aaronpowell Seems your actions are failing here
> 
> ```
> Run Install-Module -Force -Scope CurrentUser Pester -MaximumVersion '5.99.99'
> WARNING: The version '5.4.1' of module 'Pester' is currently in use. Retry the operation after closing the applications.
> No usable version of the libssl was found
> Error: Process completed with exit code 1.```
> ```

Fixed the build - the actions were all really out of date.

**Comment by aaronpowell** [comment]
Linking to #101

File: nvm.psm1 (hunk 1 above)
Comment (drekuru) [reply nvm.psm1]: Done

**Review by aaronpowell — APPROVED** [review]

**Comment by mwkcoding** [comment]
This does not seem to have been released?

**Pull request merged**